application-services/custom/fledge-export/main.go

```diff
@@ -0,0 +1,59 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/edgexfoundry/app-functions-sdk-go/appsdk"
+	"github.com/edgexfoundry/app-functions-sdk-go/pkg/transforms"
+
+	fledgeTransforms "fledge-export/pkg/transforms"
+)
+
+func main() {
+
+	// 1) First thing to do is to create an instance of the EdgeX SDK, giving it a service key
+	edgexSdk := &appsdk.AppFunctionsSDK{
+		ServiceKey: "FledgeExport", // Key used by Registry (Aka Consul)
+	}
+
+	// 2) Next, we need to initialize the SDK
+	if err := edgexSdk.Initialize(); err != nil {
+		message := fmt.Sprintf("SDK initialization failed: %v\n", err)
+		if edgexSdk.LoggingClient != nil {
+			edgexSdk.LoggingClient.Error(message)
+		} else {
+			fmt.Println(message)
+		}
+		os.Exit(-1)
+	}
+
+	// 3) Shows how to access the application's specific configuration settings.
+	fledgeEndpoint, err := edgexSdk.GetAppSettingStrings("FledgeSouthHTTPEndpoint")
+	if err != nil {
+		edgexSdk.LoggingClient.Error(err.Error())
+		os.Exit(-1)
+	}
+
+	// 4) This is our pipeline configuration, the collection of functions to
+	// execute every time an event is triggered.
+	if err := edgexSdk.SetFunctionsPipeline(
+		//transforms.NewConversion().TransformToJSON,
+		fledgeTransforms.NewConversion().TransformToFledge,
+		transforms.NewHTTPSender(fledgeEndpoint[0], "application/json", false).HTTPPost,
+	); err != nil {
+		edgexSdk.LoggingClient.Error(fmt.Sprintf("SDK SetPipeline failed: %v\n", err))
+		os.Exit(-1)
+	}
+
+	// 5) Lastly, we'll go ahead and tell the SDK to "start" and begin listening for events to trigger the pipeline.
+	err = edgexSdk.MakeItRun()
+	if err != nil {
+		edgexSdk.LoggingClient.Error("MakeItRun returned error: ", err.Error())
+		os.Exit(-1)
+	}
+
+	// Do any required cleanup here
+
+	os.Exit(0)
```

Comment:
Should be os.Exit(0)

Reply:
It is os.Exit(0).  Something else needed??